specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/virtualWan.json

```diff
@@ -1976,15 +2282,15 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualWans/{virtualWanName}/p2sVpnServerConfigurations/{p2SVpnServerConfigurationName}": {
```

Comment:
>p2sVpnServerConfigurations/{p2SVpnServerConfigurationName} [](start = 129, length = 58)

What's the plan for migrating existing resources when you delete this? Are you following the applicable steps documented at [https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html?q=deprecation](https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html?q=deprecation)?

Reply:
You will need to pay attention to the policy part of that document because there exist policy aliases for this resource type.

---
In reply to: [325936168](https://github.com/Azure/azure-rest-api-specs/pull/7238#discussion_r325936168) [](ancestors = 325936168)

Reply:
Actually, we have already contacted existing customers to delete these resources and whoever didn't delete, we have already done the cleanup on their behalf as per the deadline communicated. So, there are no existing P2SVpnServerConfiguration resources owned by customer on Production.

Reply:
@Nilambari, see my message in the email thread. Please look at the ARM deprecation guidelines (link in email). They include steps for contacting the Azure Policy team to determine the impact to customer policies, and to have us remove aliases as the properties are removed.